readers/mongodb/messages.go

```diff
@@ -109,8 +110,36 @@ func fmtCondition(chanID string, query map[string]string) *bson.D {
 			"name",
 			"protocol":
 			filter = append(filter, bson.E{Key: name, Value: value})
+		case "v":
+			fVal, err := strconv.ParseFloat(value, 64)
+			if err != nil {
+				continue
+			}
+			filter = append(filter, bson.E{Key: "value", Value: fVal})
+		case "vb":
+			bVal, err := strconv.ParseBool(value)
+			if err != nil {
+				continue
+			}
+			filter = append(filter, bson.E{Key: "bool_value", Value: bVal})
+		case "vs":
+			filter = append(filter, bson.E{Key: "string_value", Value: value})
+		case "vd":
+			filter = append(filter, bson.E{Key: "data_value", Value: value})
+		case "from":
+			fVal, err := strconv.ParseFloat(value, 64)
+			if err != nil {
+				continue
+			}
+			filter = append(filter, bson.E{Key: "time", Value: bson.M{"$gte": fVal}})
+		case "to":
+			fVal, err := strconv.ParseFloat(value, 64)
+			if err != nil {
+				continue
+			}
+			filter = append(filter, bson.E{Key: "time", Value: bson.M{"$lt": fVal}})
```

Comment:
Is there any parrticular reason to use here `$lt` instead of `$lte`. I'm asking this, since you're using `$gte` for the `from` test.

Reply:
The reason was to do it like for other readers. After to think about I prefer `>=` and `<=`. I'll do it for all readers.